3rdparty/llvm/CMakeLists.txt

```diff
@@ -14,8 +14,7 @@ if(WITH_LLVM)
 		option(LLVM_INCLUDE_TESTS OFF)
 		option(LLVM_INCLUDE_TOOLS OFF)
 		option(LLVM_INCLUDE_UTILS OFF)
-		# we globally enable ccache
-		set(LLVM_CCACHE_BUILD OFF CACHE BOOL "Set to ON for a ccache enabled build")
+		option(LLVM_CCACHE_BUILD ON)
```

Comment:
Why did you set  `LLVM_CCACHE_BUILD` to `ON`? On Linux the command line for LLVM files looks like this:
```
ccache ccache c++ ...
```
`ccache` is called twice.

Reply:
Somehow it worked fine until the changes in LLVM_CCACHE_BUILD, afterwards it started to rebuild llvm after unrelated changes.